.pre-commit-config.yaml

```diff
@@ -14,6 +14,7 @@ repos:
     rev: 21.12b0
     hooks:
       - id: black
+        language_version: python3.9
```

Comment:
set it to the lowest version of python we support (3.7)

Reply:
This should match the GitHub action.